docs/tables/oci_database_autonomous_db_metric_cpu_utilization.md

```diff
@@ -1,6 +1,6 @@
-# Table: oci_database_autonomous_database_metric_cpu_utilization_daily
+# Table: oci_database_autonomous_db_metric_cpu_utilization
```

Comment:
@ParthaI Can you please review if the table name in this document has been updated correctly